examples/create-react-app/package.json

```diff
@@ -3,8 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@lingui/react": "^3.13.2",
-    "@lingui/core": "^3.13.2",
+    "@lingui/react": "^3.14.1",
```

Comment:
Maybe, it would be better to increment the minor version for this Context improvement? **v3.15.0**. 

@semoal what do you think about that? 🙂

Reply:
Yep, it must be as a minor version; Technically we use conventional commits; If it's squashed and rebased to main with a commit title like: `feat(): add message context `
Will try to review the pull request on this morning, before merging :)

Reply:
Hi @semoal!
Have you had a chance to review this PR?